app/services/create_chat_channel_membership_service.rb

```diff
@@ -0,0 +1,14 @@
+class CreateChatChannelMembershipService < ApplicationService
+  def initialize(chat_channel, current_user)
+    @chat_channel = chat_channel
+    @user = current_user
+  end
+
+  attr_accessor :chat_channel, :user
+
+  def perform
+    membership = ChatChannelMembership.new(user_id: user.id, chat_channel_id: chat_channel.id)
```

Comment:
Since we are simply creating a `ChatChannelMembership` this doesn't feel like it needs a whole service. If we had more logic then I would be all for this but to simply use what ActiveRecord has already given us feels a bit redundant and not worth the added complexity.